.github/workflows/merge-conflicts.yml

```diff
@@ -0,0 +1,15 @@
+name: "Merge Conflicts"
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
I'd say we want this for all pull requests

Reply:
My understanding of how this action works is it runs when changes are pushed to master and checks all PRs for conflicts.

Reply:
Was reading on mobile and thought it said "pull_request" instead of "push". This is fine 👍